precompile/modules/module.go

```diff
@@ -0,0 +1,20 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package modules
+
+import (
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type Module struct {
```

Comment:
What's the rationale for making this a struct instead of an interface?

Reply:
ConfigKey, Address and Contract are all constants/singleton variables. Module is defined as a static object and should be accessed as one. Modules do not have different implementations but have only different values. So I think struct would fit better in that case. Creating a new instance for Module interfaces seems a little off to me as they supposed to be definitions rather than implementations. 

Plus we can remove all constant accesses from packages (like addresses config keys) and export only modules. Currently we export all these constants.

Reply:
Makes sense to me. Thanks for the explanation